keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -123,7 +120,7 @@ reservedSystemCPUs: "0,32,1,33,16,48"
 When `strict-cpu-reservation` is disabled:
 ```console
 # cat /var/lib/kubelet/cpu_manager_state
-{"policyName":"static","defaultCpuSet":"0-79","checksum":1241370203}
+{"policyName":"static","defaultCpuSet":"0-64","checksum":1241370203}
```

Comment:
Could you maybe please elaborate why do we need this change? It is no obvious to me

Reply:
Editorial change to align with the section below. Multiple setups have this features enabled; the two sections were copied from terminals to different setups by mistake.

Reply:
I guess is OK even though I personally found the previous example clearer.